service/src/registry.ts

```diff
@@ -460,7 +478,7 @@ export class Registry {
    * @param {Function}    done       The callback that will be called with either an Error object or
    *                                 the device twin instance.
    */
-  getTwin(deviceId: string, done: Registry.ResponseCallback): void {
+  getTwin(deviceId: string, done: TripleValueCallback<any, any>): void {
```

Comment:
not promisifying any of this?

Reply:
Done.

---
In reply to: [219359536](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r219359536) [](ancestors = 219359536)